spec/models/order_item_spec.rb

```diff
@@ -58,6 +58,8 @@
     let(:params) { {:external_url => "not.a.real/url"} }
 
     before do
+      expect(Catalog::SubmitNextOrderItem).to receive(:new).with(order_item.order_id).and_return(double(:process => nil))
```

Comment:
Should probably change this `expect` to an `allow` since it's in a before block. If you want to make a separate expectation then we would need a new `it` block somewhere.